aws/hostedzones/hosted-zones.go

```diff
@@ -0,0 +1,11 @@
+package hostedzones
+
+// HostedZones contains the Hosted Zone IDs of the cluster.
```

Comment:
I'm OK with just having the hosted zone IDs. The IDs are a bit cryptic but we have other data like that e.g the AMI ID. I think its more important to keep the TPR smaller.

I really like the separate hosted zone IDs. For now API and Etcd will be the same but that is likely to change later.